scripts/go_gen_p4_const.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/ettle/strcase"
+	"github.com/golang/protobuf/proto"
+	p4ConfigV1 "github.com/p4lang/p4runtime/go/p4/config/v1"
+	"github.com/pborman/getopt/v2"
```

Comment:
Unless there is a compelling need, we should use the same `flag` package for CLI args as in pfcpiface.